packages/client/hmi-client/src/components/workflow/ops/model-config/tera-model-config-node.vue

```diff
@@ -79,4 +103,30 @@ watch(
 	},
 	{ deep: true }
 );
+
+watch(
```

Comment:
This section is confusing me at first glance 
We have 2 watchers on the state to update our id fields
In a separate location we then have a computed on these id fields
and in a 3rd location we have a watcher on the computed which sometimes updates the state

We may be able to clean this up but im not certain on whats happening here

Reply:
Thanks for the explanation!